phpunit/LDAP/AuthLdapTest.php

```diff
@@ -2176,6 +2176,128 @@ public function testRuleRight()
         $this->assertCount(1, $gus);
     }
 
+    /**
+     * Test if rules targeting ldap criteria are working
+     *
+     * @return void
+     */
+    public function testGroupRuleRight()
+    {
+        //prepare rules
+        $rules = new \RuleRight();
+        $rules_id = $rules->add([
+            'sub_type'     => 'RuleRight',
+            'name'         => 'test ldap groupruleright',
+            'match'        => 'AND',
+            'is_active'    => 1,
+            'entities_id'  => 0,
+            'is_recursive' => 1,
+        ]);
```

Comment:
You can use `createItem()` to check that the addition is correct:
```suggestion
        $rules_id = $this->createItem(
            'RuleRight',
            [
                'sub_type'     => 'RuleRight',
                'name'         => 'test ldap groupruleright',
                'match'        => 'AND',
                'is_active'    => 1,
                'entities_id'  => 0,
                'is_recursive' => 1,
            ]
        )->getID();
```
There are also createItems(), updateItem() and updateItems()

Reply:
It wasn't the only one :)